### PR TITLE
[DO NOT REVIEW]: To change file system type from smartfs to jSmartFs

### PR DESCRIPTION
**Pull request by mkrunal**

### CODE REVIEW

**Comment by mkrunal** [comment]
Unwanted commits has been added aprt from my commits.

File: tools/nxfuse/README_SMARTFS.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# How to make smartfs user image for TizenRT
+# How to make jSmartFs user image for TizenRT
```
Comment (amogh-samsung) [inline tools/nxfuse/README_SMARTFS.md:1]: missed changes in 
tools/nxfuse/src/main.c
tools/nxfuse/src/tinyara_services.c

File: apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c [hunk 2]
```diff
@@ -324,6 +324,6 @@ int buffer_optimization_main(int argc, char *argv[])
 
 error_with_frt_fd:
 	close(frt_fd);
-	printf("SmartFs Optimized Buffer Test Example Exits\n\n");
+	printf("jSmartFs Optimized Buffer Test Example Exits\n\n");
```
Comment (amogh-samsung) [inline apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c:327]: Need to change the command name in line 279

File: apps/examples/smartfs/Makefile [hunk 3]
```diff
@@ -16,7 +16,7 @@
 #
 ###########################################################################
 
-MENUDESC = "SmartFs Test Applications"
+MENUDESC = "jSmartFs Test Applications"
```
Comment (amogh-samsung) [inline apps/examples/smartfs/Makefile:19]: The subfolder name should be changed from smartfs to jsmartfs
Here the spelling will be "jsmartfs" only

File: apps/examples/smartfs/smartfs_powercut/Kconfig_ENTRY [hunk 4]
```diff
@@ -1,3 +1,3 @@
 config ENTRY_SMARTFS_POWERCUT
-	bool "Smartfs Powercut Test example"
+	bool "jSmartFs Powercut Test example"
```
Comment (amogh-samsung) [inline apps/examples/smartfs/smartfs_powercut/Kconfig_ENTRY:2]: The TASH command for this test should be changed to "jsmartfs_powercut"

File: os/fs/driver/mtd/smart.c [hunk 5]
```diff
@@ -18,7 +18,8 @@
 /****************************************************************************
  * fs/driver/mtd/smart.c
  *
- * Sector Mapped Allocation for Really Tiny (SMART) Flash block driver.
+ * Journaling - Sector Mapped Allocation for Really Tiny (jSmartFs) Flash block driver.
```
Comment (amogh-samsung) [inline os/fs/driver/mtd/smart.c:21]: This line should not be changed.

File: os/fs/smartfs/README.txt [hunk 6]
```diff
@@ -1,17 +1,20 @@
-SMARTFS README
-^^^^^^^^^^^^^^
+jSmartFs README
+^^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
-Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
+Journaling - Sector Mapped Allocation for Really Tiny FLASH file system, jSmartFs.
+Journaling tries to provide an assurance of file system integrity during system crash.
+By writing changes to a journal quickly,we can ensure that all changes to files
```
Comment (amogh-samsung) [inline os/fs/smartfs/README.txt:7]: Space after the comma please.

File: os/fs/smartfs/smartfs_utils.c [hunk 7]
```diff
@@ -815,7 +815,7 @@ int smartfs_mount(struct smartfs_mountpt_s *fs, bool writeable)
 		goto errout;
 	}
 
-	/* Get the SMART low-level format information to validate the device has been
+	/* Get the jSmartFs low-level format information to validate the device has been
```
Comment (amogh-samsung) [inline os/fs/smartfs/smartfs_utils.c:818]: lets keep this as SMART

**Review by amogh-samsung — COMMENTED** [review]
It seems like there are places where you have missed changes.
Cannot OK this PR in public if it can be later found that changes were missed.

Apart from that some other comments.

File: apps/examples/smartfs/smartfs_powercut/Kconfig_ENTRY (hunk 4 above)
Comment (mkrunal) [reply apps/examples/smartfs/smartfs_powercut/Kconfig_ENTRY:2]: changed application name from "smartfs_powercut" to "jsmartfs_powercut" in Makefile

**Pull request closed without merging**